packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -1034,19 +1034,19 @@ describe('StripeHelper', () => {
 
   describe('findPromoCodeByCode', () => {
     it('finds a promo code', async () => {
-      const promoCode = { code: 'code1' };
+      const promotionCode = { code: 'code1' };
       sandbox
         .stub(stripeHelper.stripe.promotionCodes, 'list')
-        .resolves({ data: [promoCode] });
+        .resolves({ data: [promotionCode] });
       const actual = await stripeHelper.findPromoCodeByCode('code1');
-      assert.deepEqual(actual, promoCode);
+      assert.deepEqual(actual, promotionCode);
     });
 
     it('finds no promo code', async () => {
-      const promoCode = { code: 'code2' };
+      const promotionCode = { code: 'code2' };
       sandbox
         .stub(stripeHelper.stripe.promotionCodes, 'list')
-        .resolves({ data: [promoCode] });
+        .resolves({ data: [promotionCode] });
```

Comment:
Thanks for the these changes.  Sorry I was not clear in my original comment, which was about a mismatch between the name for the promotion code in the request payload and what the auth-server was expecting.  It was not about variable names that stayed in their scopes.  Consistency is always a 👍 though.

Reply:
original comment was clear, just wanted it to be consistent across the board and easy enough with find and replace